plugin/pulpcore/plugin/changeset/iterator.py

```diff
@@ -166,13 +169,21 @@ def _iter(self):
         Yields:
             Download: The appropriate download object.
         """
-        for artifact in ArtifactIterator(self.content):
-            if self.deferred or artifact.model.downloaded:
+        for remote_artifact in ArtifactIterator(self.content):
```

Comment:
The variable named `artifact` refers to a `RemoteArtifact` consistently in the code base, specifically naming this `remote_artifact` introduces confusion.